### PR TITLE
*: Use gogo protobuf instead of golang protobuf

### PR DESCRIPTION
**Pull request by shenli**
@siddontang @coocood @BusyJay PTAL
/cc @ngaut 
#9

### CODE REVIEW

File: generate-go.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
 cd proto
 
 echo "generate go code..."
-protoc --go_out=../go-tipb *.proto
+protoc -I.:${GOPATH}/src/github.com/gogo/protobuf:${GOPATH}/src/github.com/gogo/protobuf/protobuf --gofast_out=../go-tipb *.proto
```
Comment (siddontang) [inline generate-go.sh]: we can define a var 
```
GOGO_ROOT=${GOPATH}/src/github.com/gogo/protobuf
protoc -I. ${GOGO_ROOT}:${GOGO_ROOT}/protobuf 
```

Comment (shenli) [inline generate-go.sh]: PTAL @siddontang

**Comment by siddontang** [comment]
@BusyJay please use this to test push down first.

**Comment by shenli** [comment]
PTAL @siddontang

**Comment by siddontang** [comment]
LGTM

**Comment by BusyJay** [comment]
Tested and it works fine.

**Comment by shenli** [comment]
@coocood @siddontang @BusyJay PTAL

**Comment by coocood** [comment]
LGTM

**Pull request merged**